sapp/ui/frontend/src/index.css

```diff
@@ -17,6 +17,7 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
+  margin: 0 !important;
```

Comment:
I know very little about CSS, but I know that `!important` is usually an anti-pattern. Why is this needed?

Reply:
if we don't set it to `!important`, antd element styles rendered after index.css renders, would override the margin and set a 0.2em margin. To prevent that overriding and enforcing of margin, we could either put `style={{margin: '0'}}` to all antd code components, (4 or 3 in the issue component), or override it here by using `!important` to specify that we don't want to care for antd component css style definition corresponding to margin attribute of the component.